packages/commonwealth/client/scripts/views/pages/Community/Members/CommunityMembersPage.tsx

```diff
@@ -126,7 +126,7 @@ const CommunityMembersPage = () => {
           : true
       )
       .filter((group) =>
-        searchFilters.category === 'All'
+        searchFilters.category === 'All groups'
```

Comment:
hmm, so i actually did `"All groups" -> "All"` since it was also used in the members tab. 

I think we should revert this to "All" or maybe use both "All groups" / "All members" depending on the active tab

Reply:
Was just following the [figma](https://www.figma.com/file/ePuGFatYQI6GL72i27sVwj/Gating?node-id=1221%3A31747&mode=dev)

Which is correct? @zakhap

Reply:
Let's stick the Figma!